File: app/src/main/java/name/mizunotlt/eruditkurs/MainActivity.java
package name.mizunotlt.eruditkurs;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;

public class MainActivity extends AppCompatActivity {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }


    public void onClickNewGame(View view) {
        Intent intent = new Intent(this, GameStart.class);
        startActivity(intent);
    }


    public void onClickAbout(View view){
        Intent intent = new Intent(this, AboutActivity.class);
        startActivity(intent);
    }
}

File: TicTacToe/app/src/main/java/com/example/anew/tictactoe/MainActivity.kt
package com.example.anew.tictactoe

import android.content.Intent
import android.content.SharedPreferences
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.preference.PreferenceManager
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    var pref: SharedPreferences?= null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        pref = PreferenceManager.getDefaultSharedPreferences(this)
        val set = Intent(baseContext, Settings::class.java)
        button_set.setOnClickListener {
            startActivity(set)
        }
        val intent = Intent(baseContext, Game::class.java)
        button_main.setOnClickListener {
            if (pref?.getBoolean("sound", false)!!) {
                intent.putExtra("Flag", true)
            }
            else{
                intent.putExtra("Flag",false)
            }
            startActivity(intent)
        }
    }
}

